cdk/v2/pinterest_tag/procWorkflow.yaml

```diff
@@ -0,0 +1,209 @@
+bindings:
+  - name: EventType
+    path: ../../../constants
```

Comment:
Shouldn't this be ../../../constants/index

Reply:
index will be automatically picked up if index.js is inside that folder.